docs/apis-tools/frontend-development/03-forms/02-embed-forms/01-concepts.md

```diff
@@ -0,0 +1,148 @@
+
+See the following example form using the form viewer, and interact with it:
+
+<FormViewer schema={ policyReviewForm } />
+
+### Form playground
+
+The [form playground](https://github.com/bpmn-io/form-js/tree/develop/packages/form-js-playground) <GHIcon /> is a tool to simulate forms and explore form-js in a playful manner. It combines the [editor](#form-editor) and the [viewer](#form-viewer) with mock data input and output panels to test a form and form editor features instantly.
+
+There is also a [Camunda-flavored version of the form playground](https://github.com/camunda/form-playground) <GHIcon />, which closely resembles the form editor experience in Camunda Web and Desktop Modeler, and supports rapid development.
+
+The form playground mainly comprises the following areas:
+
+- The **component palette** to search and add components.
+- The **editor canvas**, allowing to compose a form by dragging components.
+- The **preview pane**, which shows an interactive preview of the form. The preview updates in real-time when a change happens in the editor, properties panel, or mock input data.
+- The **properties panel**, which is used to configure the properties of a component.
+- The **data input panel**, which allows to simulate the form preview using mock input data.
+- The **output panel**, which calculates and shows the current form output in real-time, based on the interactions with the preview.
+
+The input and output panel, together with the preview, come in handy to simulate the behavior of a form, and to validate or debug the configuration of one or multiple components, especially when using expressions extensively. Use the input data panel to simulate process variables, business objects, or static data used in your form.
+
+<img src={FormPlaygroundImg} alt="Camunda Forms playground" />
+
+<div style={{marginTop: '24px', marginBottom: '8px'}}>
+   <a
+      className={clsx(
+         "button button--outline button--secondary button--lg"
+      )}
+      href="https://camunda-form-playground.netlify.app/">
+      Try form playground
+   </a>
+</div>
+
+Try the form playground yourself directly on the Web—no login needed.
+
+## The form schema
+
+A form is serialized as plain JSON with a simple, flat structure to maximize flexibility and versatility. In the root, a form contains some metadata attributes. The main form is defined by a list of components, where the components carry their layout properties themselves (e.g. which row a component belongs to). This is in contrast to markup languages such as HTML, where the arrangement of the nodes determines the layout. This enables backward compatibility and compatibility with user-defined renderers. 
```

Comment:
This isn't quite true when you consider groups, right? We do have a nodal architecture.